packages/nextjs/app/builders/0x48Bf488D00FE4C83e803688ffE5532EB83E9a0ff/page.tsx

```diff
@@ -0,0 +1,142 @@
+"use client";
+
+import { useEffect, useState } from "react";
+import Image from "next/image";
+import { NextPage } from "next";
+import { useTheme } from "next-themes";
+import { FaGithub, FaLinkedin, FaTwitter } from "react-icons/fa";
+import { SiBluesky } from "react-icons/si";
+import { Address } from "~~/components/scaffold-eth";
+
+const BUILDER_ADDRESS = "0x48Bf488D00FE4C83e803688ffE5532EB83E9a0ff";
+
+const socialLinks = [
+  {
+    href: "https://github.com/nicwn",
+    label: "GitHub",
+    icon: <FaGithub className="mr-3" size={20} />,
+  },
+  {
+    href: "https://x.com/nicwn",
+    label: "X",
+    icon: <FaTwitter className="mr-3" size={20} />,
+  },
+  {
+    href: "https://linkedin.com/in/nickwang",
+    label: "LinkedIn",
+    icon: <FaLinkedin className="mr-3" size={20} />,
+  },
+  {
+    href: "https://bsky.app/profile/nicwn.bsky.social",
+    label: "Bluesky",
+    icon: <SiBluesky className="mr-3" size={20} />,
+  },
+];
+
+const ProfilePage: NextPage = () => {
+  const [isVisible, setIsVisible] = useState(false);
+  const [mounted, setMounted] = useState(false);
+  const { resolvedTheme } = useTheme();
+  const isDarkMode = resolvedTheme === "dark";
+
+  useEffect(() => {
+    setIsVisible(true);
+    setMounted(true);
+  }, []);
+
+  if (!mounted) {
+    return null;
+  }
+
+  return (
+    <div className="min-h-screen flex items-center justify-center p-4 font-sans">
+      <div
+        className={`w-full max-w-md mx-auto overflow-hidden rounded-3xl shadow-md shadow-secondary ${
+          isDarkMode ? "bg-base-100" : "bg-white"
+        } transform ${isVisible ? "translate-y-0 opacity-100" : "translate-y-8 opacity-0"} transition-all duration-700`}
+      >
+        {/* Header with pattern */}
+        <div className="relative">
+          <div className="h-36 bg-[#5DA2D5] overflow-hidden relative">
+            <div className="absolute inset-0 opacity-20">
+              <div className="absolute inset-0 bg-[radial-gradient(circle_at_50%_120%,white_1px,transparent_1px)] bg-[size:16px_16px]"></div>
+            </div>
+          </div>
+
+          {/* Avatar */}
+          <div className="absolute left-0 right-0 -bottom-16 flex justify-center">
+            <div className="p-1 bg-[#F79E02] rounded-full shadow-md shadow-secondary group">
+              <div className="p-1 bg-[#55BAB7] rounded-full overflow-hidden transform transition-all duration-300 group-hover:scale-105">
```

Comment:
You should only have hover effects for interactive elements. This hover effect misleads the user into thinking they can click it to navigate.